src/App.tsx

```diff
@@ -1,14 +1,67 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { getTodos, getUser } from './api';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [loadingTodo, setLoadingTodo] = useState(true);
+
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [loadingUser, setLoadingUser] = useState(true);
+
+  const [selectedStatus, setSelectedStatus] = useState<Status>(Status.ALL);
+
+  const [query, setQuery] = useState('');
+
+  const handleSelectedStatus = (
+    event: React.ChangeEvent<HTMLSelectElement>,
+  ) => {
+    const value = event.target.value;
+
+    if (Object.values(Status).includes(value as Status)) {
+      setSelectedStatus(value as Status);
```

Comment:
The `handleSelectedStatus` function checks if the `value` is included in `Status` values, but it directly casts `value` to `Status` without ensuring it's a valid `Status`. Consider adding a type guard or additional validation to ensure `value` is a valid `Status` before casting to prevent potential runtime errors.